packages/@lwc/babel-plugin-component/src/program.js

```diff
@@ -23,6 +23,7 @@ function exit(api) {
                 ]);
 
                 path.traverse(visitors, state);
+                dedupeImports(path);
```

Comment:
Hi! I'm new here and I've been curious about all the processes. 

I wonder if it would be nice to add a warning in the build process here flagging these imports, or maybe a eslint rule could help. In case of a eslint rule, this could even be discarded, right?

Reply:
@leobalter we already have that, this is actually to dedupe our own imports we produce along the multiple internal transforms.

Reply:
The reason we don't do that inline is because the transforms will become very trick and somewhat order aware.